src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -47,6 +47,7 @@
     <PackageReference Include="Microsoft.Build" Version="$(RefOnlyMicrosoftBuildVersion)" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(RefOnlyMicrosoftBuildTasksCoreVersion)" />
     <PackageReference Include="NuGet.ProjectModel" Version="$(RefOnlyNugetProjectModelVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
```

Comment:
This was necessary to avoid an old version of Newtonsoft.Json (transitive dependency of NuGet.ProjectModel) bringing in old runtime packages which resulted in a package downgrade error.